tests/e2e/controller_setup_test.go

```diff
@@ -60,18 +60,18 @@ func NewTestContext() (*testContext, error) {
 	// Lastly if none of them are set, it uses  $HOME/.kube/config to create the client.
 	config, err := ctrlruntime.GetConfig()
 	if err != nil {
-		return nil, fmt.Errorf("error creating the config object %v", err)
+		return nil, fmt.Errorf("Error to config object %v", err)
```

Comment:
@zdtsw is there a reason to upcase the starting letter in error messages? the recommended best practice is to have all error messages start with lower case.

Reply:
updated to lower-case